assets/scripts/app/__tests__/DateTimeRelative.test.js

```diff
@@ -19,53 +19,53 @@ describe('DateTimeRelative', () => {
   })
 
   it('renders snapshot', () => {
-    const wrapper = renderWithIntl(
+    const { asFragment } = renderWithIntl(
       <DateTimeRelative value={new Date().toISOString()} />
     )
-    expect(wrapper.asFragment()).toMatchSnapshot()
+    expect(asFragment()).toMatchSnapshot()
   })
 
   it('renders "seconds ago" string', () => {
-    const wrapper = renderWithIntl(
+    const { container } = renderWithIntl(
```

Comment:
So I like the idea of renaming these, especially considering: 
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-wrapper-as-the-variable-name-for-the-return-value-from-render

However, since 'container' is also part of the testing-library-api, this feels like a poor name which could lead to some misunderstanding: 
https://testing-library.com/docs/react-testing-library/api#container